autoload/neomake/makers/ft/c.vim

```diff
@@ -1,7 +1,9 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#c#EnabledMakers()
-    if neomake#utils#Exists('clang')
+    if neomake#utils#Exists('clang-check')
+        return ['clangcheck']
+    elseif neomake#utils#Exists('clang')
         return ['clang']
```

Comment:
I think this could just be

```
return ['clang', 'clangcheck']
```

Unless those checkers are mutually exclusive. Neomake won't run a maker that doesn't exist, so you generally don't need to check for them.